templates/schedule.html

```diff
@@ -10,8 +10,12 @@
             <div class="hero-text-wrapper">
                 <div class="hero-text">
                     <h1>Get your tickets now</h1>
-                    <p>Join us for the Matrix Conference this October. The four day conference will take place
-                        in Strasbourg, France. Talks will be streamed live and virtual attendees will be able to ask speakers questions.</p>
+                    <p>
+                        Whether you want to hear how an organization deployed
+                        Matrix at scale or learn what the engineers are doing to
+                        improve the protocol, the Matrix Conf has a track for
```

Comment:
are we doing "Matrix Conf" in formal(?) public comms now?

Reply:
I don't see a problem with it, but happy to update it to Matrix Conference if you find it more suitable :)

Reply:
Just pointing out that this would be what I think the first visible usage on the website. That almost makes it look like an error :D Branding-wise maybe we should decide to either embrace it more completely, or not and only using it informally.